CHANGELOG.md

```diff
@@ -15,7 +23,7 @@ Changelog for NeoFS Contract
 ### Fixed
 - NNS `renew` now can only be done by the domain owner
 
-### Updating from v0.15.x
+### Updating from v0.15.5
```

Comment:
Should we update `prevPatch` in `common/version.go`?

Reply:
We have the same update logic, nothing has changed.